README.md

```diff
@@ -1,3 +1,6 @@
-# BSP layer for testing OSTree with QEMU on x86_64 host
+# BSP layer for updating QEMU with OSTree
```

Comment:
```suggestion
# BSP layer for updating QEMU x86-64 guests with OSTree
```

... or something similar.

Otherwise, it sounds like we want to update qemu, the software.

Reply:
Good point, thanks.